e2e/e2etests/test_ton_deposit_and_call.go

```diff
@@ -45,14 +46,10 @@ func TestTONDepositAndCall(r *runner.E2ERunner, args []string) {
 
 	expectedDeposit := amount.Sub(depositFee)
 
+	// wait for the zrc20 balance to be updated
+	change := utils.NewExactChange(expectedDeposit.BigInt())
```

Comment:
Why do we need this variable instead of just passing delta directly to WaitForZRC20BalanceChange

Reply:
So, the `BalanceChange` type is to support both `exact` amount check and simple `increased/decreased` check.
In a few other E2E tests, we only want to check if the amount has increased or not, and there are no exact numbers there (hard to give a number).